frontend/src/lib/lemon-ui/LemonTag/LemonTag.scss

```diff
@@ -9,6 +9,7 @@
     color: var(--default);
     line-height: 1rem;
     white-space: nowrap;
+    max-width: 100%;
```

Comment:
I'm actually not sure if we want to change the behaviour or `LemonTag`-s everywhere. It might be safer to add the `w-max` CSS class to the breakdown tag explicitly.